CHANGELOG_v2.11.4.md

```
@@ -0,0 +1,211 @@
+**Previous:** `fastled/FastLED @ ^3.7.0` (caret allows any 3.x version from 3.7.0 up to but not including 4.0.0)  
+**New:** `fastled/FastLED @ 3.10.3`
+
+**Key Improvements for ESP32-S3:**
+- ✅ Optimized I2S/SPI drivers
+- ✅ Native RGBW support
+- ✅ RMT5 fixes specifically for ESP32-S3
+- ✅ Improved parallel output performance
+- ✅ Better memory management
+- ✅ Enhanced stability for high-speed LED operations
+
+**Reference:** https://github.com/FastLED/FastLED/releases/tag/3.10.3
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)